src/mono/mono/mini/mini-arm64.c

```diff
@@ -1902,6 +1917,10 @@ arg_get_storage (CallContext *ccontext, ArgInfo *ainfo)
 		case ArgVtypeInIRegs:
 		case ArgInIReg:
 			return &ccontext->gregs [ainfo->reg];
+		case ArgSwiftSelf:
+			return &ccontext->gregs [PARAM_REGS + 1 + ARMREG_R20 - CTX_REGS_OFFSET];
```

Comment:
I don't see the point of `ARMREG_R20 - CTX_REGS_OFFSET` subtraction, we are hardcoding R20 here. Seems confusing. Why not just store at `PARAM_REGS + 1` and at `PARAM_REGS + 1 + 1` below.

Reply:
The idea was to indicate the self and error registers on arm64. The use of `PARAM_REGS + 1` and `PARAM_REGS + 1 + 1` is not descriptive in terms of which registers are being used. I agree that your suggestion is simpler and not confusing.

Reply:
Updated.